extension/js/content_scripts/webmail/setup-webmail-content-script.ts

```diff
@@ -414,7 +414,7 @@ export const contentScriptSetupIfVacant = async (webmailSpecific: WebmailSpecifi
       }
       warningMsg =
         expirationText +
-        `To receive the latest keys, please ensure that you can connect to your corporate network either through VPN or in person and reload Gmail.<br/>` +
+        `To receive the latest keys, please ensure that you are connected to your corporate network (or through VPN) and you have entered your Flowcrypt passphrase. Then reload Gmail.<br/>` +
```

Comment:
let's remove `you` in `and you have entered`, to remove repetition, and also use `FlowCrypt` instead of `Flowcrypt`:
> ... and have entered your FlowCrypt passphrase ...